arangod/Cluster/ClusterInfo.h

```diff
@@ -269,9 +270,9 @@ class CollectionInfoCurrent {
   /// @brief local helper to return a map to boolean
   //////////////////////////////////////////////////////////////////////////////
 
-  template<typename T>
-  std::unordered_map<ShardID, bool> getFlag(T const& name) const {
-    std::unordered_map<ShardID, bool> m;
+  [[nodiscard]] containers::FlatHashMap<ShardID, bool> getFlag(
+      std::string const& name) const {
```

Comment:
Can we use `std::string_view` for `name` as well here?

Reply:
Same here